webapp/src/components/sidebar_header/sidebar_header.tsx

```diff
@@ -0,0 +1,25 @@
+import React, {PureComponent, ReactElement} from 'react';
+
+import {Theme} from 'mattermost-redux/types/preferences';
+
+import SidebarButtons from '../sidebar_buttons';
+
+interface SidebarHeaderProps{
+    show: boolean,
+    theme: Theme
+}
+
+export default class SidebarHeader extends PureComponent<SidebarHeaderProps> {
+    render(): ReactElement | null {
+        if (!this.props.show) {
+            return null;
```

Comment:
Can we convert this into a function component in this PR?

```suggestion
export default function SidebarHeader(props: SidebarHeaderProps): React.ReactNode {
    if (!props.show) {
        return null;
```

Reply:
@Willy-Wakam This actually isn't necessary for this PR